src/openssl.c

```diff
@@ -92,6 +92,20 @@ write_bn(unsigned char *buf, const BIGNUM *bn, int bn_bytes)
 }
 #endif
 
+static inline void
+_libssh2_swap_bytes(unsigned char *buf, unsigned long len)
```

Comment:
Unfortunately, the new APIs require BIGNUMs to be in host byte order instead of network byte order.